src/include/sound/volume_routines.rb

```diff
@@ -224,6 +226,8 @@ def GlobExists(glob)
       out = Convert.to_map(
         SCR.Execute(
           path(".target.bash_output"),
+          # cannot escape it here as escaped glob return without asterisk expansion,
+          # but we provide input
           Builtins.sformat("echo -n %1", glob)
```

Comment:
Could we use [Dir.glob](https://ruby-doc.org/core-2.5.1/Dir.html#method-c-glob) here?

Reply:
Well, it would be the best solution, but much bigger impact, so higher risk of bugs.